### ISSUE TITLE
Make liked tracks playable

### ISSUE THREAD

**Issue by dann-merlin** [dann-merlin/Spotipyne#4]
Make liked tracks playable.
This is probably a bit harder and more complex that it sounds. The spotify web api has no native support to just play a song in the context of 'saved tracks'. This basically means one would have to manually send all the context_uris of the songs on the saved_tracks one would want to play.
It gets really complicated, when shuffle comes into play. Because then on a song change, one would need to check if shuffle is enabled and if that is the case, play a random song in the saved tracks list. When shuffle is turned back off again, one would need to keep playing in the order described by the saved tracks list. Also, I don't think when supplying a custom list of context_uris (song uris), spotify will respect the repeat setting, so this will have to be emulated as well.
:(
For a start, it will probably be a nice feature to be able to just play one track.

One could maybe kind of look over at the code of spotify-tui. AFAIK they kind of already do this, but only send the few tracks, that are being displayed on the screen at the time (at least the last time I checked).

**Comment by Pacman99** [comment]
I just tried out the rfct_library-overview branch and it shows the liked songs playlist, so thats really exciting. 

You're right this is a lot harder than I thought and I see what you mean looking at the spotipy documentation.

In the start_playback function, it accepts an array of uri's, could you just pass the saved tracks list to that and not pass a context_uri?

And I have one kinda hacky idea: maybe spotipyne could maintain a synced playlist of the liked tracks like "Liked Songs" or something.

**Comment by dann-merlin** [comment]
> In the start_playback function, it accepts an array of uri's, could you just pass the saved tracks list to that and not pass a context_uri?

Yes, that will probably work, but I don't know how well that would work with the most likely HUGE number of uris of all the songs.

> And I have one kinda hacky idea: maybe spotipyne could maintain a synced playlist of the liked tracks like "Liked Songs" or something.

Yeah, I actually do have a playlist like that myself, but that playlist would be visible in the other clients as well.

> I just tried out the rfct_library-overview branch and it shows the liked songs playlist, so thats really exciting.

Yes, I did work on some issues this weekend and pushed them to master. Also a "play random" button and some backend stuff.
